docs/pipelines.md

```diff
@@ -850,6 +851,8 @@ spec:
       timeout: "0h1m30s"
 ```
 
+### Pipelines in Pipelines
```

Comment:
Please mention that this is a feature in preview mode - something similar to - https://github.com/tektoncd/pipeline/pull/2650/files#diff-778db8afe943ee8b2b7f3acf5ec1eb68d7bf0c926a1ddb46866125d1a4c0ce7fR533